README.md

```diff
@@ -50,64 +56,89 @@ Binaries will be placed in `target/debug` under the project's root directory.
 
 # Build File Systems
 
-Please refer to [snapfaas-images](https://www.github.com/princeton-sns/snapfaas-images)
+In `rootfs` directory, the script `mk_rtimage.sh` will create the necessary file
+system image. The runtimes are either `nodejs` or `python3`. An example basic
+usage is
+
+```bash
+./mk_rtimage.sh python3 hello-python3.ext4
```

Comment:
I think it's better to just use `python3.ext4`. The name `hello-python3.ext4` is misleading in that it reads as if the image contains an application already.